### PR TITLE
create recipe step

### PR DESCRIPTION
**Pull request by tuanbpl-0873**
https://edu-redmine.sun-asterisk.vn/issues/10638

### CODE REVIEW

File: app/Http/Controllers/Frontend/CreateRecipeController.php [hunk 1]
```diff
@@ -85,10 +85,119 @@ public function createFirstForm(CreateRecipeFirstRequest $request)
...
+        $thisStep = $this->recipe->findCookingStep($id, $stepId);
+        switch ($request->submit_step) {
+            case 'next_step':
+                $nextStep = $stepId + 1;
+                $dataStepInfo = [
+                    'step_number' => $request->step_number,
+                    'name' => $request->name,
+                    'time' => $request->time,
+                    'note' => $request->note,
+                    'content' =>$request->content
```
Comment (framgiaciv3) [inline app/Http/Controllers/Frontend/CreateRecipeController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoSpaceAfterDoubleArrow</br>**Severity**: error</br>**File**: app/Http/Controllers/Frontend/CreateRecipeController.php</br>**Line**: 141</br>Expected 1 space between double arrow and "$request"; 0 found

Comment (framgiaciv3) [inline app/Http/Controllers/Frontend/CreateRecipeController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/Frontend/CreateRecipeController.php</br>**Line**: 141</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/Frontend/CreateRecipeController.php [hunk 2]
```diff
@@ -85,10 +85,119 @@ public function createFirstForm(CreateRecipeFirstRequest $request)
...
+                ]);
+            break;
+        
+            case 'next_form':
+                $dataStepInfo = [
+                    'step_number' => $request->step_number,
+                    'name' => $request->name,
+                    'time' => $request->time,
+                    'note' => $request->note,
+                    'content' =>$request->content
```
Comment (framgiaciv3) [inline app/Http/Controllers/Frontend/CreateRecipeController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoSpaceAfterDoubleArrow</br>**Severity**: error</br>**File**: app/Http/Controllers/Frontend/CreateRecipeController.php</br>**Line**: 160</br>Expected 1 space between double arrow and "$request"; 0 found

Comment (framgiaciv3) [inline app/Http/Controllers/Frontend/CreateRecipeController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/Frontend/CreateRecipeController.php</br>**Line**: 160</br>Each line in an array declaration must end in a comma

File: app/Http/Middleware/isLogin.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Auth;
```
Comment (framgiaciv3) [inline app/Http/Middleware/isLogin.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Classes.ValidClassName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Middleware/isLogin.php</br>**Line**: 8</br>Class name "isLogin" is not in PascalCase format

**Comment by tuanbpl-0873** [comment]
ready

File: app/Http/Controllers/Frontend/CreateRecipeController.php [hunk 4]
```diff
@@ -85,10 +85,119 @@ public function createFirstForm(CreateRecipeFirstRequest $request)
...
+                    'time' => $request->time,
+                    'note' => $request->note,
+                    'content' => $request->content,
+                ];
+                if (isset($thisStep)) {
+                    $this->recipe->updateCookingStep($id, $stepId, $dataStepInfo);
+                } else {
+                    $recipe->cookingStep()->create($dataStepInfo);
+                }
+                return redirect()->route('form.step', [
```
Comment (chinn01-0830) [inline app/Http/Controllers/Frontend/CreateRecipeController.php]: convention

File: app/Http/Controllers/Frontend/CreateRecipeController.php [hunk 5]
```diff
@@ -85,10 +85,119 @@ public function createFirstForm(CreateRecipeFirstRequest $request)
...
+                    'time' => $request->time,
+                    'note' => $request->note,
+                    'content' => $request->content,
+                ];
+                if (isset($thisStep)) {
+                    $this->recipe->updateCookingStep($id, $stepId, $dataStepInfo);
+                } else {
+                    $recipe->cookingStep()->create($dataStepInfo);
+                }
+                return redirect()->route('form.categories', [
```
Comment (chinn01-0830) [inline app/Http/Controllers/Frontend/CreateRecipeController.php]: convention

File: resources/views/frontend/create-recipe/ingredient-form.blade.php [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+<link rel="stylesheet" href="{{ asset('bower_components/bootstrap/dist/css/bootstrap.min.css') }}">
+@endsection
+
+@section('content')
+
+<div class="wrap clearfix">
+    <!--breadcrumbs-->
+    <nav role="navigation" class="breadcrumbs">
+        <ul>
+            <li><a href="{{ route('home') }}" title="Home">'{{ __('Home') }}</a></li>
```
Comment (chinn01-0830) [inline resources/views/frontend/create-recipe/ingredient-form.blade.php]: i18n title :v

**Pull request merged**